PhpUnit/DefinitionIsChildOfConstraint.php

```diff
@@ -46,7 +47,13 @@ public function toString()
 
     private function evaluateDefinitionIsDecorator(Definition $definition, $returnResult)
     {
-        if (!($definition instanceof DefinitionDecorator)) {
+        if (!class_exists(ChildDefinition::class) && !$definition instanceof DefinitionDecorator) {
```

Comment:
could just be `if (!$definition instanceof ChildDefinition && !$definition instance of DefinitionDecorator)`